src/scss/components/_view_wrapper.scss

```
@@ -71,7 +71,7 @@ article.#{$tdp-css-prefix}-view-wrapper {
     }
   }
 
-  > a.view-help {
+  > a.view-help, div.view-help {
```

Could we simply use the following?

```suggestion
  > .view-help {
```